disnake/embeds.py

```diff
@@ -267,12 +271,20 @@ def from_dict(cls, data: EmbedData) -> Self:
         self.timestamp = utils.parse_time(data.get("timestamp"))
 
         self._thumbnail = data.get("thumbnail")
+        if self._thumbnail and (thumbnail_flags := self._thumbnail.get("flags")):
+            self._thumbnail["flags"] = EmbedMediaFlags._from_value(thumbnail_flags)  # type: ignore
+
         self._video = data.get("video")
         self._provider = data.get("provider")
         self._author = data.get("author")
+
         self._image = data.get("image")
+        if self._image and (image_flags := self._image.get("flags")):
+            self._image["flags"] = EmbedMediaFlags._from_value(image_flags)  # type: ignore
+
```

Comment:
I don't like how this is done but I don't see much alternatives

Reply:
This also no longer seems to be respected in serialisation.